db/schema.rb

```diff
@@ -128,5 +122,6 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
   end
 
+  add_foreign_key "phases", "users"
   add_foreign_key "project_leads", "users"
```

Comment:
their must be more missing foriegn keys i am sure

Reply:
Yeah I look into the scheme and find one more FK to be added. Added also.